### ISSUE TITLE
NTR: Nipples position high (Chest, respiratory & breathing)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7792]
New term request Nipples position high (elthat_160123181232): Mamilla; papillae  UMLS: 4231433

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000229
- label: High-set nipples
- parents: [Abnormal nipple morphology (HP:0004404)](https://hpo.jax.org/app/browse/term/HP:0004404)
- pmid: 
- def: Placement of the nipples at a higher than normal location.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7792
- synonyms:
